### PR TITLE
Added the 'no_reboot` flag to the ActivateRequest message.

### PR DESCRIPTION
**Pull request by tomek-US**
If this flag is set to 'False' then the Target will configure itself
to boot next time with the requested version of the OS but will _not_
restart immediatelly.

### CODE REVIEW

File: os/os.proto [hunk 1]
```diff
@@ -250,11 +251,18 @@ message InstallError {
...
   string version = 1;
   // For dual Supervisors setting this flag instructs the Target to perform the
   // action on the Standby Supervisor.
   bool standby_supervisor = 2;
+  // If set to 'True' the Target will initiate the reboot process immediatelly
+  // after changing the next bootable OS version.
+  // If set to 'False' a separate action to reboot the Target and start using
+  // the activated OS version is required. This action CAN be executing
+  // the gNOI.system.Reboot() RPC.
+  bool activate_and_reboot = 3;
```
Comment (samribeiro) [inline os/os.proto:265]: The inverse logic should be used here in order to keep the same behaviour with backward compatibility. Something like "no_reboot" or "activate_only" (proto3 defaults boolean to false when omitted). Where by default the Activate request will activate and reboot the target, unless this flag is set in which only activation will happen, but no reboot. This allows previous and actual versioned client and target implementations to coexist without a fundamental behaviour change of the RPC like it is being proposed here.

File: os/os.proto [hunk 2]
```diff
@@ -7,7 +7,7 @@ import "github.com/openconfig/gnoi/types/types.proto";
 
 option go_package = "github.com/openconfig/gnoi/os";
 
-option (types.gnoi_version) = "0.1.0";
+option (types.gnoi_version) = "0.2.0";
```
Comment (samribeiro) [inline os/os.proto:10]: assuming the suggestion below, I am thinking this would be a micro bump and opposed to minor.

File: os/os.proto (hunk 1 above)
Comment (tomek-US) [reply os/os.proto]: Keeping backward compatibility sounds like good idea ;-) Done.

File: os/os.proto (hunk 2 above)
Comment (tomek-US) [reply os/os.proto]: Done

**Review by samribeiro — APPROVED** [review]
thanks!

**Comment by samribeiro** [comment]
Probably worth regenerating the go code for this proto.

**Comment by tomek-US** [comment]
Thx!

The second commit already includes re-generated Go files.

**Pull request merged**